lib/ruby_lsp/test_helper.rb

```diff
@@ -11,6 +11,7 @@ class TestError < StandardError; end
     extend T::Helpers
 
     requires_ancestor { Kernel }
+    requires_ancestor { Minitest::Test }
```

Comment:
for `flunk`

Reply:
This would be tying our test helpers to only Minitest, but people can use whatever framework to write add-ons.

Let's not use `flunk` and instead just raise a specific error.